aquadoggo/src/lib.rs

```diff
@@ -59,3 +60,11 @@ fn init() {
         let _ = env_logger::builder().is_test(true).try_init();
     }
 }
+
+fn info_or_print(message: &str) {
```

Comment:
A doc-string for this would be nice at it appears on root level and it took me a second to understand what this is for (I like it!)